src/webrtc/call.ts

```diff
@@ -2247,7 +2241,12 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
         }
     }
 
-    private queueCandidate(content: RTCIceCandidate): void {
+    /**
+     * Queue a candidate to be sent
+     * @param content The candidate to queue up, or null if candidates have finished being generated
+     *                and end-of-candidates should be signalled
+     */
+    private queueCandidate(content: RTCIceCandidate | null): void {
```

Comment:
I wonder if we could avoid the step with `null` and set `candidatesEnded` right at the start?

Reply:
Not sure what you mean here - at the start of what?

Reply:
I was thinking we would set `candidatesEnded = true` right after receiving the last candidate and avoid calling `queueCandidate()` with null though since that might require a bit more work (if even possible) and you're on holiday, we can go with this as is

Reply:
oh I see - we need to schedule it to be sent too though, so this seemed the easiest way of re-using that code.